.github/workflows/ci.yml

```
@@ -697,6 +697,10 @@ jobs:
           ABFS_CONTAINER: ${{ vars.AZURE_ABFS_HIERARCHICAL_CONTAINER }}
           ABFS_ACCOUNT: ${{ vars.AZURE_ABFS_HIERARCHICAL_ACCOUNT }}
           ABFS_ACCESS_KEY: ${{ secrets.AZURE_ABFS_HIERARCHICAL_ACCESS_KEY }}
+          SNOWFLAKE_PASSWORD: "secret"
+          SNOWFLAKE_ACCOUNT_NAME: "account"
+          SNOWFLAKE_CATALOG_S3_ACCESS_KEY: "access"
+          SNOWFLAKE_CATALOG_S3_SECRET_ACCESS_KEY: "secret"
```

Let's change these to match the format of other vars/secrets

```suggestion
          SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
          SNOWFLAKE_ACCOUNT_NAME: ${{ vars.SNOWFLAKE_ACCOUNT_NAME }}
          SNOWFLAKE_CATALOG_S3_ACCESS_KEY_ID: ${{ vars.SNOWFLAKE_S3_ACCESS_KEY_ID }}
          SNOWFLAKE_CATALOG_S3_SECRET_ACCESS_KEY: ${{ secrets.SNOWFLAKE_S3_SECRET_ACCESS_KEY }}
```